tests/rbd_mirror/test-add-remove-osd-with-mirroring.py

```diff
@@ -0,0 +1,188 @@
+from ceph.parallel import parallel
+from tests.rbd_mirror.rbd_mirror_utils import execute_dynamic, rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_add_remove_osd_with_mirroring(rbd_mirror, pool_type, **kw):
```

Comment:
method doc is missing here also please add `log.info `for test steps if possible.

Reply:
Done.